fdbt-dev/data/netexData/carnetReturn.xml

```diff
@@ -88,7 +90,10 @@
             <ValueSet version="1.0" id="Branding" classOfValues="Branding">
               <Name>Operator Branding</Name>
               <values>
-                <Branding version="1.0" id="op:BLAC@brand"/>
+                <Branding version="1.0" id="op:BLAC@brand">
```

Comment:
This is also a common change, how come they're all Blackpool Transport?

Reply:
Blackpool is operator that we use to run the tests that generate these netex files for comparison so all the data brings through the Blackpool values.